src/test/java/rv/atqc/gttesting/archex/AbstractPage.java

```diff
@@ -4,14 +4,18 @@
 
 import org.openqa.selenium.JavascriptExecutor;
 import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
 import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
 
 import rv.atqc.gttesting.archex.helpers.Acting;
 
 public abstract class AbstractPage<T extends AbstractPage<T>> {
 
 	protected WebDriver driver;
 	protected JavascriptExecutor jse;
+	private WebDriverWait wait;
```

Comment:
You need to read my comments for Roman or merge dev after Roman will merge changes with waiters